modules/effects/arrow-effects-rx2-extensions/src/test/kotlin/arrow/effects/FlowableKTests.kt

```diff
@@ -85,10 +79,9 @@ class FlowableKTests : UnitSpec() {
     //testLaws(AsyncLaws.laws(FlowableK.asyncMissing(), EQ(), EQ()))
     //testLaws(AsyncLaws.laws(FlowableK.asyncMissing(), EQ(), EQ()))
 
-    testLaws(
-      FoldableLaws.laws(FlowableK.foldable(), { FlowableK.just(it) }, Eq.any()),
-      TraverseLaws.laws(FlowableK.traverse(), FlowableK.functor(), { FlowableK.just(it) }, EQ())
-    )
+    testLaws(FoldableLaws.laws(FlowableK.foldable(), { FlowableK.just(it) }, Eq.any()))
+
+    testLaws(TraverseLaws.laws(FlowableK.traverse(), FlowableK.functor(), { FlowableK.just(it) }, EQ()))
```

Comment:
These look wrong. TraverseLaws imply FoldableLaws, so line 82 could be just deleted.